h2o-docs/src/product/sphinxext/docscrape.py

```diff
@@ -369,7 +369,7 @@ def _str_index(self):
         idx = self['index']
         out = []
         out += ['.. index:: %s' % idx.get('default','')]
-        for section, references in idx.iteritems():
```

Comment:
Since we have an update in documentation, we should not forgot to check if nightly release and documentation is also without any issues

Reply:
Good point!